pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -409,11 +416,11 @@ func (s *Scheduler) trySchedule(ctx context.Context, p *corev1.Pod) error {
 		if err == nil {
 			return nil
 		}
-		// We should only relax the pod's requirements when the error is not a reserved offering error because the pod may be
+		// We should only relax the pod's requirements when the error is not a reserved offering error or DRA error because the pod may be
```

Comment:
I wouldn't add it on in this sentence because the only part that applies is that we "shouldn't relax the pod's requirements". I'd either add another paragraph at the end or use a separate if statement with its own brief blurb.